.eslintoutputrc

```diff
@@ -8,7 +8,8 @@
     {
       "name": "junit",
       "output": "file",
-      "path": "tmp/junit.xml"
+      "path": "tmp/junit.xml",
+      "id": "myJunit"
     }
   ]
 }
```

Comment:
didnt notice this before but maybe add an empty line at the end of this file

Reply:
Good point, not sure how that got lost.